examples/buildconfig/docker-compose.yml

```diff
@@ -3,5 +3,6 @@ version: "2"
 services:
     foo:
         build: "./build"
+        image: "localhost:5000/vagrant/foo:latest"
         command: "sleep 3600"
```

Comment:
this might be a useful reference:
https://github.com/openshift/origin/blob/master/pkg/build/api/v1/types.go#L463-L487